builtin/mainmenu/dlg_delete_world.lua

```diff
@@ -17,39 +17,37 @@
 
 
 local function delete_world_formspec(dialogdata)
-
 	local retval =
-		"size[11.5,4.5,true]" ..
-		"label[2,2;" ..
+		"size[10,2.5,true]" ..
+		"label[0.5,0.5;" ..
 		fgettext("Delete World \"$1\"?", dialogdata.delete_name) .. "]" ..
-		"button[3.25,3.5;2.5,0.5;world_delete_confirm;" .. fgettext("Delete") .. "]" ..
-		"button[5.75,3.5;2.5,0.5;world_delete_cancel;" .. fgettext("Cancel") .. "]"
+		"button[0.5,1.5;2.5,0.5;world_delete_confirm;" ..
+		minetest.colorize("#FFFF00", fgettext("Delete")) .. "]" ..
```

Comment:
Does yellow offer enough contrast to not confuse it accidentally with white? Perhaps orange?

Reply:
Yellow does seem too close to white.

Reply:
I've tested with bright red, doesn't work. Will try bright orange.

Reply:
Nope, I've no idea. Please see my latest comment.